vars/common-vars.yml

```diff
@@ -17,6 +17,10 @@ socok8s_libvirtuuid: "{{ socok8s_workspace }}/libvirt.uuid"
 suse_osh_registry_location: "docker.io"
 suse_openstack_image_version: "rocky-opensuse_15-20190613"
 suse_infra_image_version: "opensuse_15-20190613"
+suse_osh_openstack_release: "rocky"
+
+obs_mirror: "{{ lookup('env','SOCOK8S_OPENSUSE_MIRROR') | default('https://download.opensuse.org', true) }}"
+ibs_mirror: http://ibs-mirror.prv.suse.net
```

Comment:
Hm, where is this coming from? I think this already in stable/1.0  (https://github.com/SUSE-Cloud/socok8s/pull/569/files)

Reply:
Hm, looks like botched rebase/cherry-pick ...